src/billing/template/containerTemplate.jsx

```diff
@@ -0,0 +1,131 @@
+/* @flow */
```

Comment:
Is there anything different here from the checkout overlay? If so can we have that as a common function between the two components?

Reply:
Looks like this is still duplicated?

Reply:
oh, the UI of billing address popup is slightly different with the checkout so I reuse whatever I can.